java/classfile/src/org/netbeans/modules/classfile/package.html

```diff
@@ -73,7 +73,7 @@ <h2>Examples</h2>
 <h2>Related Documentation</h2>
 
 <ul>
-  <li><A HREF="http://java.sun.com/docs/books/jls/index.html">Java Virtual Machine Specification, Second Edition</a>
+  <li><A HREF="hhttps://docs.oracle.com/javase/specs/jvms/se6/html/VMSpecTOC.doc.html">Java Virtual Machine Specification, Second Edition</a>
```

Comment:
There is an extra `h` in `http`. Consider using `https://docs.oracle.com/javase/specs/jvms/se8/html/index.html`

Reply:
well seems the scanner for wrong url didn't issue an error. Will not for later check